Wancer/Wancer/MeContactView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MeContactView.swift
+//  Wancer
+//
+//  Created by Ankit Amonkar on 3/25/24.
+//
+ 
+import SwiftUI
+import SwiftData
+import ContactsUI
+import Foundation
+import Contacts
+ 
+struct MeContactView: View {
+    @Environment(\.modelContext) private var modelContext
+    @State private var phoneNumber: String = ""
+    @Query private var users: [User] = []
+    var body: some View {
+        NavigationView {
+            Form {
+                Section(header: Text("Personal Information")) {
+                    TextField("Phone Number", text: Binding<String>(
+                        get: { self.phoneNumber },
+                        set: { newValue in  self.phoneNumber = newValue.filter {"0123456789".contains($0) }
+                        }
+                    ))
+                    .keyboardType(.numberPad)
+                }
+                Section {
+                    Button("Save") {
+                        checkUser(phoneNumber: phoneNumber, users: users)
+                    }
+                }
+            }
+            .navigationBarTitle("Your Contact Info")
+        }
```

Comment:
You can use .onAppear { ... } and copy over the fetchallcontactsandinsertintodatabase (idc).

Reply:
Ok, so if I do that, I would need to copy over the whole file from Contacts page since fetchallcontactsandinsertintodatabase relies on fetchAllContacts and insertContactsIntoDatabase. Then in doing this, it might remove ContactsPage completely. Thoughts?